framework/console/controllers/FixtureController.php

```diff
@@ -415,7 +415,15 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            // strip $fixturesPath from $fixture's full path
+            $relativeName = str_replace($fixturesPath . "/", "", $fixture);
+            // get fixtures's directory
+            $relativeDir = dirname($relativeName) == '.' ? '' : dirname($relativeName) . '/';
+            // get fixture name relatively to $fixturesPath
+            $relativeName = $relativeDir . basename($fixture, 'Fixture.php');
+            // normalize path to unix format
+            $relativeName = str_replace("\\", "/", $relativeName);
```

Comment:
Shouldn't it be done to `$fixture` prior to replacements right before `// strip $fixturesPath from $fixture's full path` line?